apps/client-server/src/app/websites/implementations/toyhouse/models/toyhouse-file-submission.ts

```diff
@@ -0,0 +1,140 @@
+import { BooleanField, DescriptionField, RatingField, SelectField, TagField, TextField, TitleField } from '@postybirb/form-builder';
+import { DescriptionType, DescriptionValue, SubmissionRating } from '@postybirb/types';
+import { BaseWebsiteOptions } from '../../../models/base-website-options';
+import { ToyhouseAccountData } from './toyhouse-account-data';
+
+const ToyhouseMaturityOptions = [
+  { value: SubmissionRating.GENERAL, label: 'No Sexual Content' },
+  { value: SubmissionRating.MATURE, label: 'Mild Sexual Content' },
+  { value: SubmissionRating.ADULT, label: 'Explicit Sexual Content' },
+];
+
+const ToyhousePrivacyOptions = [
+  { value: '0', label: 'Full-Size' },
+  { value: '1', label: 'Watermarked' },
+  { value: '2', label: 'Thumbnail' },
+  { value: '3', label: 'Hidden' },
+];
+
+const ToyhouseWatermarkOptions = [
+  { value: '1', label: 'Default (Center)' },
+  { value: '2', label: 'Default (Stretch)' },
+  { value: '3', label: 'Default (Tile)' },
+  // Custom watermarks are not supported at this time.
+];
+
+export class ToyhouseFileSubmission extends BaseWebsiteOptions {
+  @TitleField({
+    required: false,
+    hidden: true,
+  })
+  title: never;
+
+  @TagField({
+    hidden: true,
+  })
+  tags: never;
+
+
+  @RatingField({
+    options: ToyhouseMaturityOptions,
+    required: true,
+    section: 'common', order: 1
+  })
+  rating: SubmissionRating;
+
+  @BooleanField({
+    label: 'nudity',
+    span: 4,
+    section: 'common', order: 2
+  })
+  nudity: boolean;
+
+  @BooleanField({
+    label: 'gore',
+    span: 4,
+    section: 'common', order: 2
+  })
+  gore: boolean;
+
+  @BooleanField({
+    label: 'sensitiveContent',
+    span: 4,
+    section: 'common', order: 2
```

Comment:
All these that you have set to 'common' should just be 'website' not set (as they default to website)

Reply:
I don't really have time for quite a bit to fix the UX here, but I guess you guys can figure it out. Basically content warning has a dependency on sensitive content (which is different in toyhouse). I implemented it so it shows up when you select sensitive content, but now that if I move it it jumps around weirdly. We can make it so its always visible I guess, and that checkbox input doesn't matter if sensitive content is present. Works that way for global rules.

Heres what i mean by jumps around with the changes:

https://github.com/user-attachments/assets/d2b0b765-aea5-4c7c-a393-1f1b75d0ee7f